delira/training/predictor.py

```diff
@@ -169,7 +169,7 @@ def predict_data_mgr(self, datamgr, batchsize=None, metrics=None,
             a dictionary containing all metrics of the current batch
 
         """
-
+        print(metric_keys)
```

Comment:
Print statement should be removed :)

Reply:
I noticed this too and just removed them before your comments :)